packages/sanity/src/core/form/components/ActivateOnFocus/ActivateOnFocus.tsx

```diff
@@ -59,13 +61,19 @@ export function ActivateOnFocus(props: ActivateOnFocusProps) {
 
   const msg = useMemo(() => {
     const isTouch = isTouchDevice()
-    let activateVerb = isTouch ? 'Tap' : 'Click'
+    let activateVerb = isTouch
+      ? t('inputs.portable-text.activate-on-focus.tap-verb')
+      : t('inputs.portable-text.activate-on-focus.click-verb')
     if (focused && !isTouch) {
-      activateVerb += ' or press space'
+      activateVerb += t('inputs.portable-text.activate-on-focus.focused-verb')
     }
```

Comment:
This should probably be two separate strings with a context passed in as you can't just concatenate words like this in every language. Similar to [this](https://github.com/sanity-io/sanity/pull/5140/files) PR